helm/values.yaml

```diff
@@ -109,3 +109,24 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+# configuration for OpenTelemetry Collector sidecar
+# https://opentelemetry.io/docs/collector/
+otel:
+  enabled: false
+  # https://opentelemetry.io/docs/collector/configuration/
+  config:
+  # default otel-collector-sidecar will listen to otlp over http
+  containers:
+    - name: otel-collector-sidecar
+      image: otel/opentelemetry-collector-contrib
```

Comment:
Open question: should the image be limited to some version tag to be compatible with the `otel` library?

Reply:
Yes thats probably wise, I'll pin it